testsuite/integration-arquillian/tests/other/webauthn/src/main/java/org/keycloak/testsuite/webauthn/pages/fragments/ContentAlert.java

```diff
@@ -32,17 +31,25 @@
 
 /**
  * @author Vaclav Muzikar <vmuzikar@redhat.com>
+ *
+ * Page fragments seem not to be working after migration from CGlib to ByteBuddy in Graphene
  */
 public class ContentAlert {
-    private static final String ROOT_ID = "//ul[@data-testid='alerts']/li[1]";
+    private static final String ROOT_ID = "//ul[@data-testid='alerts']/li[1]//div";
```

Comment:
Why is there suddenly a new `div` element here?

Reply:
@jonkoops It's not new. Look at https://github.com/keycloak/keycloak/pull/29779/files/f0d1bbbc43f44cde05ede561262c0b8a50c092ed#diff-f8e9b02d58aa1b5e991c53ddd11e557c6fcf07a992bf67a032994f167ea834adL40

It just localizes the previously called "root" element. 

The thing is that after upgrading Arquillian Graphene, the Page fragments functionally seem not working (as mentioned in the Javadoc above). IMHO, it's not worth waiting for a fix in Graphene as we use it only at this place - so I removed the page fragment.